### ISSUE TITLE
init.lua:523: attempt to concatenate a nil value

### ISSUE THREAD

**Issue by Bastrabun** [minetest-mods/personal_log#4]
Not sure what exactly lead to the error.

```
2021-10-08 23:11:01: ACTION[Main]: Server: Shutting down
2021-10-08 23:11:13: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'personal_log' in callback on_playerReceiveFields(): /home/mtlive1/.minetest/mods/personal_log/init.lua:523: attempt to concatenate a nil value
2021-10-08 23:11:13: ERROR[Main]: stack traceback:
2021-10-08 23:11:13: ERROR[Main]: 	/home/mtlive1/.minetest/mods/personal_log/init.lua:523: in function 'make_personal_log_formspec'
2021-10-08 23:11:13: ERROR[Main]: 	/home/mtlive1/.minetest/mods/personal_log/init.lua:685: in function 'update_callback'
2021-10-08 23:11:13: ERROR[Main]: 	/home/mtlive1/.minetest/mods/personal_log/init.lua:577: in function 'on_player_receive_fields'
2021-10-08 23:11:13: ERROR[Main]: 	/home/mtlive1/.minetest/mods/personal_log/init.lua:684: in function 'func'
2021-10-08 23:11:13: ERROR[Main]: 	...inetest_live/bin/../builtin/profiler/instrumentation.lua:106: in function <...inetest_live/bin/../builtin/profiler/instrumentation.lua:100>
2021-10-08 23:11:13: ERROR[Main]: 	...ve1/5.4.1/Minetest_live/bin/../builtin/game/register.lua:422: in function <...ve1/5.4.1/Minetest_live/bin/../builtin/game/register.lua:406>
```

Any plans on converting the save file to json or the like?

**Comment by Bastrabun** [comment]
For the record, this happened again. 5.5.0 with current MTG

**Comment by SmallJoker** [comment]
Fixed as of 8b63913.
